pom.xml

```diff
@@ -33,17 +33,20 @@
         <dice-provider-artifact-id>dice-provider</dice-provider-artifact-id>
         <x509-provider-artifact-id>x509-provider</x509-provider-artifact-id>
 
-        <iot-device-client-version>1.33.1</iot-device-client-version>
```

Comment:
Looks like you bumped some versions here (for test purposes?) that you'll need to revert